lib/Runtime/Language/SourceTextModuleRecord.cpp

```diff
@@ -903,6 +898,13 @@ namespace Js
             scriptContext->GetDebugContext()->RegisterFunction(this->rootFunction->GetFunctionBody(), nullptr);
         }
 #endif
+        if (childrenModuleSet != nullptr)
+        {
+            childrenModuleSet->EachValue([=](SourceTextModuleRecord* childModuleRecord)
```

Comment:
I know this was copied from before, but since this doesnt need to capture _anything_ in the outer scope, much less _everything_, could we remove the `[=]` to do so?

Reply:
It needs to access the childrenModuleSet within the EachValue function - which in turn uses it to populate the childModuleRecord.

Reply:
Except in the lambda body, only `childModuleRecord` is referenced, which is passed as a parameter to the lambda. Nothing from the lexical scope is used. With that being said, I just looked again at http://en.cppreference.com/w/cpp/language/lambda and it looks like the capture-all groups still only capture what is required in the lambda body, so there ends up being no actual runtime effect here -- my bad.